### ISSUE TITLE
"r2c --help" not working

### ISSUE THREAD

**Issue by jmrgonza** [uwefladrich/rdy2cpl#19]
I am getting an import error when I try to run the `r2c --help` command, related with the `namcouple.factory` module:

> ModuleNotFoundError: No module named 'rdy2cpl.namcouple.factory'; 'rdy2cpl.namcouple' is not a package

I was able to fix it by creating empty `__init__.py` files in the `rdy2cpl` and `rdy2cpl/namcouple` directories.

**Comment by jmrgonza** [comment]
I cloned the repository and checked out the most recent main branch (commit f60e2618d2d6ab407ba988f99e5026d3da27524c).
I installed it in a conda environment running `pip install .`

**Comment by jmrgonza** [comment]
Well, I was using a pre-existing environment which I had setup with pyoasis, mpi4py and other stuff. If it is working for you with a clean installation, it is probably some issue with my environment. Does it also work for you if you run `r2c --help` from other directory different to the package root? It is possible that, when you are in that directory, python can find the modules using relative paths instead of using the files installed in the conda environment? In any case, please do not waste time on this if it is working for you.

**Comment by uwefladrich** [comment]
>[...] Does it also work for you if you run `r2c --help` from other directory different to the package root? It is possible that, when you are in that directory, python can find the modules using relative paths instead of using the files installed in the conda environment? [...]

Yes, it does work from other directories as well. That is also how it is used in ECE4.

I will keep an eye on these installation issues! As I said, I do not exclude that there is something with the module auto-detection that is not perfect.

Thanks for reporting!
